gordo_components/builder/mlflow_utils.py

```diff
@@ -257,7 +257,12 @@ def get_batch_kwargs(metadata: dict) -> dict:
 
     metric_list: List[Metric] = list()
 
-    # Parse cross-validation metrics
+    # Parse cross-validation scoring metrics
+    if metadata["model"].get("cross-validation", {}).get("splits", None):
+        splits = metadata["model"]["cross-validation"]["splits"]
+        param_list += get_params(splits, splits.keys())
```

Comment:
Is this a thing we're going to have to make a habit of? As in, each time a new parameter is added to the metadata we'll have to update this too? :fearful: 

Also, after working on removing watchman and using the server instead, I've found working in this module, as it relates to the `metadata` quite difficult/tedious to debug. IMO, we're making too many assumptions about the structure of the metadata; not sure this is the right place to handle it, but making mention of it anyway. :sweat_smile:

Reply:
Yeah, I agree this is very gross and terrible. It was a concession made when the metadata couldn't be parsed in a more general way for mlflow, and one that I think needs to change.

Reply:
When we upgrade to Python 3.8, at least we can do: 
```
if splits := metadata["model"].get("cross-validation", {}).get("splits", None):
```
:heart_eyes: